src/highdicom/sr/templates.py

```diff
@@ -1935,20 +2983,427 @@ def __init__(
         item.ContentSequence.append(container_item)
         self.append(item)
 
+    def _find_measurement_groups(self) -> Sequence[ContainerContentItem]:
+        root_item = self[0]
+        imaging_measurement_items = find_content_items(
+            root_item,
+            name=codes.DCM.ImagingMeasurements,
+            value_type=ValueTypeValues.CONTAINER
+        )
+        if len(imaging_measurement_items) == 0:
+            return []
+        return find_content_items(
+            imaging_measurement_items[0],
+            name=codes.DCM.MeasurementGroup,
+            value_type=ValueTypeValues.CONTAINER
+        )
+
+    @classmethod
+    def from_sequence(cls, sequence: Sequence[Dataset]) -> 'MeasurementReport':
+        """Construct instance from a sequence of datasets.
```

Comment:
"instance" is maybe a confusing word to use here?

Reply:
I agree. That term could be confusing. What would you suggest? How about "Construct instance **of the class**."

Reply:
`Construct object` or `Construct MeasurementReport`?

Reply:
Addressed via 505ea884ab767ff872222c35f207f564874bc17d